crates/sui-types/src/dynamic_field.rs

```diff
@@ -253,11 +254,9 @@ where
 {
     let parent: SuiAddress = parent.into();
     let k_tag_bytes = bcs::to_bytes(key_type_tag)?;
-    tracing::trace!(
+    debug!(
```

Comment:
Btw for the trace->debug change we probably don't want to turn it on in main branch.

Reply:
Yep, will revert this before landing, but leave now for runs

Reply:
Done